.github/workflows/wheels.yml

```diff
@@ -85,6 +101,15 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
 
+      # Comment out when numpy/scipy dependencies are on pypi
+      - name: Set pypi index for python dev
+        if: matrix.python == 310
+        run: |
+          echo "EXTRA_CIBW_ENVIRONMENT=PIP_EXTRA_INDEX_URL=https://pypi.anaconda.org/scipy-wheels-nightly/simple PIP_PRE=true" >> $GITHUB_ENV
+
+          # Do not use the dev version of pandas
+          echo "PANDAS_VERSION_SPECIFIER=<1.4" >> $GITHUB_ENV
```

Comment:
I am trying to think of a good way to generalize this for python 3.11. I think we will end up with som